public/app/features/dashboard/components/ShareModal/ShareExport.tsx

```diff
@@ -40,6 +39,8 @@ export class ShareExport extends PureComponent<Props, State> {
     const { dashboard } = this.props;
     const { shareExternally } = this.state;
 
+    reportInteraction('dashboards_sharing_export_download_json_clicked', { externally: shareExternally });
```

Comment:
```suggestion
    reportInteraction('dashboards_sharing_export_save_json_clicked', { externally: shareExternally });
```
Or was it intentional to use download even though the word `Save` is used in the UI?

Reply:
Good catch, I tried to use UI names and I missed this one.